plugins/TestRunner/templates/matomo-tests.yml.twig

```diff
@@ -98,6 +98,8 @@ jobs:
 {% if hasSubmodules %}
           submodules: true
 {% endif %}
+      - name: Install package ripgrep
+        run: sudo apt-get install ripgrep
```

Comment:
Wouldn't it have been smarter to add that to the github action in `matomo-org/github-action-tests`, that way it' wouldn't have been required to add it to every single plugin

Reply:
Yeah, not sure why it didn't cross my mind then.

@AltamashShaikh keen to move it?

Reply:
@sgiehl I wasn't aware if that was possible, I would have added it there definitely and it wold have saved alot of time for me, @michalkleiner I can create a PR to move this in `github-action-tests`